services/blockchain-indexer/methods/indexer.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const {
+	triggerAccountUpdates,
+	indexAllDelegateAccounts,
+	cacheLegacyAccountInfo,
+	indexGenesisAccounts,
+	indexNewBlocks,
+	indexMissingBlocks,
+	getMissingBlocksListByRange,
+} = require('./controllers/indexer');
+
+module.exports = [
+	{
+		name: 'indexMissingBlocks',
+		controller: indexMissingBlocks,
+		params: {},
+	},
+	{
+		name: 'indexNewBlocks',
+		controller: indexNewBlocks,
+		params: {
+			block: { optional: false, type: 'any' },
+		},
```

Comment:
Kindly consider renaming the method to `indexNewBlocks` and pass the `blockHeight` or `blockID` as the param instead of the complete block. The `indexNewBlock` method should be responsible to fetch the block from the blockchain-coordinator.